scripts/clang-format.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/sh
+
+if test x"$1" = x; then 
+	FILES=`git ls-files | grep '\.[ch]$'`
+fi
+if test x"$1" = xchanged; then 
+	FILES=`git diff --name-only | grep '\.[ch]$'`
+fi
+
+for i in $FILES; do
+	clang-format-7 $i > /tmp/tmp-clang.c
+	mv /tmp/tmp-clang.c $i
```

Comment:
`clang-format` supports a `-i` option for in place updates. This shouldn't be necessary.

See: https://clang.llvm.org/docs/ClangFormat.html